### DISCUSSION TITLE
Color palettes

### DISCUSSION THREAD

**Discussion by khyde** [NOAA-EDAB/ecodata#196] [General]
@sgaichas asked for some of my color palettes for mapped images. Before I go through some options, I wanted to first figure out the best format. I have full 256 RGB arrays as .csv files. Is that something you can use and convert to R? An example file is attached. This is the color palette I use for the chlorophyll maps. If that works, I will send a few others along that we can consider as options. 
[cp_navy_gold.csv](https://github.com/user-attachments/files/18131931/cp_navy_gold.csv)
![cp_navy_gold-colorbox](https://github.com/user-attachments/assets/fd4c8ef6-c063-402f-a0c8-0f0cb511d7cb)

**Comment by andybeet** [comment]
yes, this is great.

```
kimColor <- readr::read_csv(here::here("data/cp_navy_gold.csv"),show_col_types=F)
palette <- rgb(kimColor$R,kimColor$G,kimColor$B,maxColorValue = 255)
ggplot2::ggplot(ggplot2::faithfuld, ggplot2::aes(waiting, eruptions, fill = density)) +
  ggplot2::geom_tile() + 
  ggplot2::scale_fill_gradientn(colors = palette)
```
produces this:
![image](https://github.com/user-attachments/assets/85f4252d-aef5-41f4-85ce-028b72db61a2)

Might be worth thinking of a utility graphing package or a more general utils package for all of us to use. Then we can have all of these csv files available as data objects

**Comment by khyde** [comment]
Should I put the csv's in the Utilities repo?

**Comment by andybeet** [comment]
i'm not sure that is the right place. but i'm also not sure i really understand what the focus is of this repo. I thought it was was for pulling and processing climatological, chlorophyll, ERDDAP stuff.

Maybe until we decide how and where we want to house and access common resources, google drive is best? Not sure though.

**Comment by khyde** [comment]
I'm going to add color palettes to my presentation for the January 6th SeaSide chat and if there is time we can discuss access to shared resources.
